.gitignore

```diff
@@ -1,3 +1,5 @@
 /target
 **/*.rs.bk
-Cargo.lock
\ No newline at end of file
+Cargo.lock
```

Comment:
Should we really ignore `Cargo.lock`?

Reply:
I am not quite sure about this, usually the good rule is - ignore Cargo.lock for libs, version control Cargo.lock for binaries